src/Sylius/Bundle/AdminBundle/Resources/config/grids/taxon.yml

```diff
@@ -0,0 +1,14 @@
+sylius_grid:
+    grids:
+        sylius_admin_taxon:
+            driver:
+                name: doctrine/orm
+                options:
+                    class: "%sylius.model.taxon.class%"
+                    repository:
+                        method: createQueryBuilderByTranslation
```

Comment:
It is not really "by translation", in only joins them. I think we should use simple convention for all grid list query builders and call them ``createListQueryBuilder``. Otherwise it is quite difficult to come up with a sensible name for these methods. They will join a lot of stuff.